### ISSUE TITLE
make check fails ?

### ISSUE THREAD

**Issue by mingodad** [msharov/ustl#20]
Hello !
I'm testing ustl and when I execute "make check" it ends with "make: *** [bvt/run] Error 1" and it seems that the problem is on bvt/Module.mk:
```
# The correct output of a bvt is stored in bvtXX.std
# When the bvt runs, its output is compared to .std
#
bvt/run:	${bvt/BVTS}
	@echo "Running build verification tests:"
	@for i in ${bvt/BVTS}; do	\
	    BVT="bvt/$$(basename $$i)";	\
	    echo "Running $$i";		\
	    ./$$i < $$BVT.cc &> $$i.out;	\   <<<<<<< here nothing goes to $$.out , the "&>" should be only ">"  ???
	    diff $$BVT.std $$i.out && rm -f $$i.out; \
	done
```

**Comment by msharov** [comment]
&> redirects both stdout and stderr to the specified file. It is not an error.
You'll have to specify more detail on which test fails. Please run the failing test manually (.o/bvt/bvtXX) and note the output. If it crashes, I would appreciate a backtrace; you can get one by compiling with debugging information enabled, running the failing test in gdb and issuing the "where" command when it crashes.

**Comment by mingodad** [comment]
I'm running on ubuntu 14.04 and I removed the " && rm -f $$i.out"" after the diff command and could see that all "$$i.out" had size 0, then changing the "&>" to only ">" it then I had content on $$i.out.

Now that you mention that you've expected stderr & stdout together I changed that line to:
```
 ./$$i < $$BVT.cc > $$i.out 2>&1;	\
```
Now the tests pass, I also tested on a raspberry pi with te same results before and after the mentioned changes.

It seems that on actual debian/ubuntu distributions tha shell/bash do not behave as you expect.

Cheers !
